.gitignore

```diff
@@ -1,8 +1,18 @@
+# PLFA specific items to ignore
 !cabal.project.ci.local
 !data/legacy/19.08.zip
 !data/legacy/20.07.zip
 !plfa.pdf
```

Comment:
These exclusions should be below their respective inclusions.

Reply:
Done.